src/App.jsx

```diff
@@ -9,36 +10,18 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState('tab-1');
```

Comment:
Better to use `tabs[0].id` instead of hard-coded value `tab-1` provides more flexible setting.

```suggestion
  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
```

Reply:
Agree, thanks!